git-webkit setup: Prefer local user.name and email over global git setting

serakeri opened the pull request:
#### b92d2c69a13c3fa27bf6b92fabaeebc1475e7328
<pre>
git-webkit setup: Prefer local user.name and email over global git setting
<a href="https://bugs.webkit.org/show_bug.cgi?id=243156">https://bugs.webkit.org/show_bug.cgi?id=243156</a>

Reviewed by Jonathan Bedard.

* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py:
(Setup.git):

Canonical link: <a href="https://commits.webkit.org/253573@main">https://commits.webkit.org/253573@main</a>
</pre>

JonWBedard reviewed (approved):
Can't think of why I did this the other way around in the first place...this seems like the more correct precedence.

serakeri committed:
git-webkit setup: Prefer local user.name and email over global git setting
https://bugs.webkit.org/show_bug.cgi?id=243156

Reviewed by Jonathan Bedard.

* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py:
(Setup.git):

Canonical link: https://commits.webkit.org/253573@main
